CGMBLEKit Example/ViewController.swift

```diff
@@ -32,6 +32,66 @@ class ViewController: UIViewController, TransmitterDelegate, UITextFieldDelegate
         stayConnectedSwitch.isOn = AppDelegate.sharedDelegate.transmitter?.stayConnected ?? false
 
         transmitterIDField.text = AppDelegate.sharedDelegate.transmitter?.ID
+
+        titleLabel.isUserInteractionEnabled = true
```

Comment:
This is just an example app... but this is what UIButton is for.

Reply:
Fair call. I was trying to keep it simple, but will change.